write-fonts/Cargo.toml

```diff
@@ -10,6 +10,7 @@ categories = ["text-processing", "parsing", "graphics"]
 
 [features]
 default = ["dot2"]
```

Comment:
I don't think we want dot2 by default do we? [I know it's not changed here]

Reply:
not in the long-term, but for now it's nice to not need to pass an extra flag at the command line if I want to write out the graph in a test or similar 🤷